.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: File a bug report
+labels: ["Type: bug"]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug, including details regarding any error messages, version, and platform.
+      description: Please include what you expected.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component
+      description: Which component is principally affected?
```

Comment:
It would be nice if several values could be selected at once, but I guess that isn't possible?

Reply:
It is possible, see https://github.com/apache/arrow/pull/14688

Reply:
Addressed in https://github.com/apache/arrow/pull/14688.